src/components/Amass/Amass.tsx

```diff
@@ -0,0 +1,97 @@
+import { Button, Stack, TextInput } from "@mantine/core";
+import { useForm } from "@mantine/form";
+import { useCallback, useState } from "react";
+import { CommandHelper } from "../../utils/CommandHelper";
+import ConsoleWrapper from "../ConsoleWrapper/ConsoleWrapper";
+import { UserGuide } from "../UserGuide/UserGuide";
```

Comment:
You imported outdated component. It should be RenderComponent 
```suggestion
import { RenderComponent } from "../UserGuide/UserGuide";
```